Misc/NEWS.d/next/Library/2019-02-18-12-45-33.bpo-35537.xshtGJ.rst

```diff
@@ -0,0 +1 @@
+subprocess.py now uses posix_spawnp instead of posix_spawn.
```

Comment:
You can enhance the entry using better formatting, and please avoid ".py" when you mention a module. User use "import subprocess", they don't access *directly* to subprocess.py. NEWS entries are for end users.

```suggestion
The :mod:`subprocess` now uses :func:`os.posix_spawnp` instead of :func:`os.posix_spawn`. 
```